tests/deploy/e2e/test_timers.py

```diff
@@ -72,10 +72,7 @@ async def test_notification_mechanism(agent, environment, clienthelper, client,
 
     def is_approx(rid: str, seconds: int) -> None:
         time = tm.resource_timers[rid].next_scheduled_time
-        assert abs(time - last_deploy_time_approx - timedelta(seconds=seconds)) < timedelta(milliseconds=100)
-
-    def is_disabled(rid) -> None:
-        assert rid not in tm.resource_timers or tm.resource_timers[rid].when is None
+        assert abs(time - last_deploy_time_approx - timedelta(seconds=seconds)) < timedelta(milliseconds=200)
```

Comment:
Was `timedelta(milliseconds=100)` causing too much flakiness ?

Reply:
on jenkins it was